source/_posts/Video-js-5-11-0-Prelease.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Video.js 5.11.0 Prelease
+author:
+  name: Gary Katsevman
+  github: gkatsev
+date: 2016-07-22 15:46:51
+tags:
+   - news
+   - releases
+   - version
+---
+Today sees the prerelease of version 5.11.0. I wanted to take a moment to talk about some of the changes, additions, and known issues.
+
+This is a pre-release only. It's available on npm under the `next` tag and also available on the CDN under the fully qualified version number: `//vjs.zencdn.net/5.11.0/video.js`. It'll stay in pre-release state for around a week or more to make sure that there aren't any major glaring bugs, so, please give it a shot and [open issues](https://github.com/videojs/video.js/issues/new) if you find anything.
```

Comment:
> to make sure there aren't any major glaring bugs,

If we're going to keep both adjectives we need a comma between them, but I think we could drop "major"